### PR TITLE
Hotfix national load

### LINKED ISSUE #71: Gap filling of load time series does not work for the default configuration
In the current default configuration, load time series from 2016 is used for all countries in Europe. We have load data of Albania (ALB) only starting from 2017 and thus, `euro-calliope 1.0` used to use 2017 data for Albania.

The current gap filling method should do the same (as I understand it), but it does not do it. Therefore, `develop` is currently broken with the following error message: 

``AssertionError: Not enough load data for country ALB``

### PR DESCRIPTION
**Pull request by brynpickering**
This should do the trick to fix #71. It is not a change in the data processing, only in the assertion that the data processing has done what was expected.

This will perform slightly differently to euro-calliope v1.0 as it does not attempt to fill 29th of Feb with data from another year for Albania (it will get it by interpolation), whereas I think the old one filled that day with the values from the day before/after. This 29th of Feb issue will be dealt with in #66 more generally.

### CODE REVIEW

File: scripts/national_load.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def _get_index_of_missing_data(series, model_year):
     return series[(series < 0) | series.isna()][str(model_year)].index
 
 
-def _handle_feb_29th(this_year, next_avail_year, missing_timesteps):
+def _ignore_feb_29th(this_year, next_avail_year, missing_timesteps):
     """ Ignore February 29th if next available year doesn't have that data available """
     if (pd.Period(freq='Y', year=this_year).is_leap_year and
```
Comment (timtroendle) [inline scripts/national_load.py:63]: You could also use `calendar` here:

`calendar.isleap(year)`

File: scripts/national_load.py [hunk 2]
```diff
@@ -78,31 +78,30 @@ def fill_missing_data_in_country(country_series, model_year, acceptable_gap_hour
...
                 year_order.append(abs(year - model_year) + 0.1)
             else:
                 year_order.append(year - model_year)
         return year_order
-    preferred_order_of_years_with_data = list(years_with_data.sort_values(key=_get_year_order, ascending=False))
+    preferred_order_of_years_with_data = list(years_with_data.sort_values(key=_give_older_years_lower_priority, ascending=False))
 
-    while country_series.loc[all_missing_timesteps].isnull().sum() > acceptable_gap_hours:
+    _missing_timesteps = all_missing_timesteps.copy()
+    while len(_missing_timesteps) > acceptable_gap_hours:
```
Comment (timtroendle) [inline scripts/national_load.py:92]: Rather combine the two conditions:

```Python
while (len(_missing_timesteps) > acceptable_gap_hours) & (preferred_order_of_years_with_data != []):
    do something
```    

This should also work but isn't more readable I think:

```Python
while (len(_missing_timesteps) > acceptable_gap_hours) & preferred_order_of_years_with_data:
    do something
```

**Review by timtroendle — APPROVED** [review]
I actually have no clue why this works. But if you say it does, I am happy with it. :)

File: scripts/national_load.py (hunk 1 above)
Comment (brynpickering) [reply scripts/national_load.py:63]: neat

**Pull request merged**